src/Microsoft.ML.TensorFlow/TensorflowCatalog.cs

```diff
@@ -102,7 +103,16 @@ public static class TensorflowCatalog
         /// <param name="tensorFlowModel">The pre-loaded TensorFlow model.</param>
         public static TensorFlowEstimator TensorFlow(this TransformsCatalog catalog,
```

Comment:
>TensorFlow [](start = 42, length = 10)

So this I find pretty confusing. Do we create estimators via *this* method, or do we work through the model object?